classes/shaman.lua

```diff
@@ -291,18 +399,33 @@ local function loadOptions(self)
     local tidalWavesSoDBuff = 432041;
     local tidalWavesSoDTalent = 432233;
 
+    --Cataclysm
+    local lavaBurstCata = 51505;
+    local lavaSurgeTalentCata = 77756;

```

Comment:
`lavaBurstCata` and `lavaSurgeTalentCata` are not used anymore in `loadOptions`, since recent commit.

Reply:
Removed in 773860f39b174ce23bab992e81d957455314d387.